pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -7,6 +7,8 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	rbac "k8s.io/kubernetes/pkg/apis/rbac"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
```

Comment:
I'd prefer to avoid this import since the bootstrap policy doesn't logically require an API type dependency.